src/validate_oNHM_daily_output.R

```diff
@@ -65,15 +65,15 @@ validate_oNHM_daily_output <- function(var, fn, test_date, data_nc, hruids, time
     filter(DOY == lubridate::yday(today)) %>%
     rename(hruid = nhru) %>%
     arrange(hruid) %>%
-    select(hruid,`90%`)
-  
+    select(hruid,`90%`) %>%
+    
   # add max value for comparison 150% of the max value/90th value
-  var_quantile_df$max_value <- var_quantile_df$`90%`*1.5
-  var_quantile_df$today <- data_nc
+    mutate(max_value <- `90%`*1.5) %>%
+    mutate(today <- data_nc)
```

Comment:
This might work but could be dangerous. You'll want to use `=` and not `<-` when inside a dplyr fxn because you are not making an object in the environment, but instead, passing arguments through a function.

Reply:
that's probably why i'm seeing the strange naming!